action.yml

```diff
@@ -28,6 +28,18 @@ inputs:
   customParameters:
     required: false
     description: 'Extra parameters to configure the Unity editor run.'
+  createCheck:
+    required: false
+    default: 'false'
+    description: 'Creates a check with the Test Results'
```

Comment:
Should we put a bit more details here or additional entry in the documentation to make it clear what this actually does? Do we expect users of this feature to know what's a _check_?

Reply:
Yeah, we would need more documentation, but I think it can be done in a separate PR in the documentation repo. 
This [action.yml is used for metadata](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions) that I don't expect the average user will see, so I think we can keep it brief as long as we have complete docs on the [unity-test-runner docs page](https://game.ci/docs/github/test-runner).
I'll raise the PR for the docs when I have some more time.